tests/forte/data/ontology/test_outputs/ft/onto/race_qa_installed_ontology.py

```diff
@@ -1,5 +1,5 @@
 # ***automatically_generated***
-# ***source json:tests/forte/data/ontology/test_specs/race_qa_onto_installed.json***
+# ***source json:tests/forte/data/ontology/tests/test_specs/race_qa_onto_installed.json***
```

Comment:
This path is actually wrong, the `json:tests/forte/data/ontology/test_specs/race_qa_onto_installed.json` is correct